contents/blog/how-to-run-transparent-company.md

```diff
@@ -0,0 +1,135 @@
+---
+date: 2022-07-26
+title: "How to run a transparent startup"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+---
+
+There really hasn't been a single situation where sharing pretty much everything we do hasn't been to our company's advantage.
+
+Most companies seem to share when necessary but not by default. I hope this piece changes that - as a startup, you've done something wrong if you don't trust your own team.
+
+Being realistic, to help this most people - this is mainly written about _internal_ transparency... and it's startup-focussed too, since that's what I know.
+
+If you want to gain even more advantages here, I would encourage you though to share this stuff by default externally unless there's a good reason not to.
+
+There are probably some startups where this doesn't work. Perhaps in design led companies, where control might be a competitive advantage and is more important than autonomy, or in companies that aren't managed consistently - you can't share a non-existent pay structure, for example (and, yes, it is hard work to implement something like that from very early, but we felt that work was worth it to prevent unfairness and employee churn).
+
+## Why this matters
```

Comment:
```suggestion
## Why transparency matters
```

Reading on from the previous sentence makes this feel unclear. Are we talking about pay structures in particular, or transparency in general.